docs/conf.py

```diff
@@ -3,67 +3,6 @@
 # For the full list of built-in configuration values, see the documentation:
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
 
-import os
-import sys
-import inspect
+from documenteer.conf.guide import *
 
-sys.path.insert(0, os.path.abspath(".."))
-
-# -- Project information -----------------------------------------------------
-# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
-
-project = "uranography"
-copyright = "2023, Fermi Research Alliance, LLC."
-author = "Eric Neilsen"
-release = "v0.1.0"
-
-# -- General configuration ---------------------------------------------------
-# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
-
-extensions = [
-    "sphinx.ext.autodoc",
-    "sphinx.ext.doctest",
-    "sphinx.ext.linkcode",
-    "sphinx.ext.autosummary",
-    "sphinx-prompt",
-    "numpydoc",
-]
-
-autosummary_generate = True
-source_suffix = ".rst"
-master_doc = "source/index"
-
-templates_path = ["_templates"]
-exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
-
-
-# -- Options for HTML output -------------------------------------------------
-# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
-
-html_theme = "classic"
-html_static_path = ["_static"]
-
-
-def linkcode_resolve(domain, info):
```

Comment:
Do the links to other documentation still work without this code?

Reply:
Do you mean a link to bokeh? Or something else? It would be good if you can confirm it looks as you would expect (https://uranography.lsst.io/v/PREOPS-3517/index.html)
I am not sure what to put in for the "URL" below though, because I wasn't sure what this was trying to do. It looked more like it was trying to make the API code link to the source code, which is not in the documenteer capabilities yet (but I think is planned to be).

Reply:
(actually links to specific bokeh classes where they were explicitly linked do work .. like to the bokeh figure docs, but I'm not sure if that's what you mean)

Reply:
I misremembered what these links were for. The code you removed is supposed to be used by sphinx to generate links to the source code (in github) from the online documentation of what that code does.

Reply:
There was a different way to do this that @jonathansick pointed me to, that should link all of the documentation to the source (a copy of the source files, per class), which matches the file in each build.

Reply:
And yes, moving index.html into the docs directory directly (instead of a subdirectory) was what was necessary to have the docs build using documenter's package-docs build command. The update to conf.py and addition of documenteer.toml was to match the suggested setup for documenteer/the rubin users guide. 
Documenteer[guide] itself adds different formatting for the final pages generated, including things like adding the rubin logo.